ext/websocket/lib.rs

```diff
@@ -426,6 +471,54 @@ pub async fn op_ws_send(
   Ok(())
 }
 
+#[op]
+pub async fn op_ws_send_string(
+  state: Rc<RefCell<OpState>>,
+  rid: ResourceId,
+  text: String,
+) -> Result<(), AnyError> {
+  let resource = state
+    .borrow_mut()
+    .resource_table
+    .get::<WsStreamResource>(rid)?;
+  resource.send(Message::Text(text)).await?;
+  Ok(())
+}
+
+#[op]
+pub async fn op_ws_send_binary(
+  state: Rc<RefCell<OpState>>,
+  rid: ResourceId,
+  data: ZeroCopyBuf,
+) -> Result<(), AnyError> {
+  let resource = state
+    .borrow_mut()
+    .resource_table
+    .get::<WsStreamResource>(rid)?;
+  resource.send(Message::Binary(data.to_vec())).await?;
+  Ok(())
+}
+
+#[op]
+pub fn op_ws_try_send_string(
+  state: &mut OpState,
+  rid: ResourceId,
+  text: String,
```

Comment:
Would it make sense to make this op an `#[op(v8)]`, pass text in as a `serde_v8::Value` and then use the `v8::String::write_utf8` method to write directly into the sink? (I'm not sure if that's actually possible either.)

Reply:
Nevermind, just noticed that the `Message` type actually requires a String so this is as optimal as one can get I guess.